www/store/utils/slice-generator.js

```diff
@@ -47,19 +47,27 @@ export function generateReducers(schema) {
     reducers[`set${capitalize(k)}`] = (state, action) => {
       const { _id, [`${v}Ids`]: refIds } = action.payload;
 
-      state[_id][k] = refIds;
+      state[_id] = { ...state[_id], [k]: refIds };
     };
 
     reducers[`create${dispName}`] = (state, action) => {
       const { _id, [`${v}Id`]: refId } = action.payload;
 
-      state[_id][k].push(refId);
+      if (Array.isArray(state[_id]?.[k])) {
+        state[_id][k].push(refId);
+      } else {
+        state[_id][k] = [refId];
+      }
```

Comment:
This and the other changes to this file prevent an error that could occur if a parent document like a meeting doesn't exist in the store when child documents are being set, created, or deleted resulting in a `Cannot set property of undefined` error.